code/Makefile

```diff
@@ -492,16 +492,17 @@ deploy: analysis graphs convertAnalyzed packagedeps docs tex doxygen ##@Deploy G
 
 # Removes all generated folders.
 $(filter $(CLEAN_GF_PREFIX)%, $(CLEAN_FOLDERS)): $(CLEAN_GF_PREFIX)%:
-	- rm -rf "./$*/"
+	rm -rf "./$*/"
```

Comment:
why this change?

Reply:
It's _almost completely_ redundant. Passing `-f` `rm` already suppresses the majority of errors, which is what `-` would do for that Makefile line. The errors that `rm -f` return all have to do with a system that we somehow were trying to edit, but which we were stopped by the OS for some reason (e.g., we accidentally tried to delete System32 or any other read-only file, the local working folders became read-only [permission loss], etc.). I think we would want to see those errors, because it would mean something has gone very seriously wrong.